src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -66,6 +67,13 @@ public void init() throws ServletException {
   // TODO: Check and see if lectureURL is already in database and if it is valid.
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Entity lectureEntity = createLectureEntity(request);
```

Comment:
Can this be moved to after the isPresent() check? Only create it if it is going to be added

Reply:
fixed